circus-train-aws/src/main/java/com/hotels/bdp/circustrain/aws/AWSConstants.java

```diff
@@ -23,7 +23,7 @@
   public static final String FS_PROTOCOL_S3 = "s3";
   public static final String FS_PROTOCOL_S3N = "s3n";
   public static final String FS_PROTOCOL_S3A = "s3a";
-  public static final String[] S3_FS_PROTOCOLS = { FS_PROTOCOL_S3, FS_PROTOCOL_S3N, FS_PROTOCOL_S3A };
+  static final String[] S3_FS_PROTOCOLS = { FS_PROTOCOL_S3, FS_PROTOCOL_S3N, FS_PROTOCOL_S3A };
```

Comment:
This is not used CT big query or some other external module?

Reply:
Is searching on GitHub for "S3_FS_PROTOCOLS" enough? I searched for it in circus-train-bigquery and housekeeping and there were no matches for that.

Reply:
yeah that should be ok, thanks!